src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -32,28 +46,41 @@ public CDSSubsystem() {
   public void CDSBeltWheelControllerToggle(boolean reverse) {
     if (reverse) {
       CDSWheelControllerOne.getSparkMax().set(Constants.CDSWheelControllerSpeed);
-      SmartDashboard.putString("CDS Wheel Controller Direction", "Reverse");
-      SmartDashboard.putNumber("CDS Wheel Controller Speed", -Constants.CDSWheelControllerSpeed);
+      CDSWheelControllerDirection.setString("Reverse");
+
+      //SmartDashboard.putString("CDS Wheel Controller Direction", "Reverse");
+      //SmartDashboard.putNumber("CDS Wheel Controller Speed", -Constants.CDSWheelControllerSpeed);
       
       CDSBeltController.getSparkMax().set(-Constants.CDSBeltSpeed);
       CDSBeltController.setIdleMode(IdleMode.kBrake);
-      SmartDashboard.putString("CDS Belt Direction", "Reverse");
-      SmartDashboard.putNumber("CDS Belt Speed", Constants.CDSBeltSpeed);
+      CDSBeltControllerDirection.setString("Reverse");
+
+      //SmartDashboard.putString("CDS Belt Direction", "Reverse");
+      //SmartDashboard.putNumber("CDS Belt Speed", Constants.CDSBeltSpeed);
     } else {
       CDSWheelControllerOne.getSparkMax().set(Constants.CDSWheelControllerSpeed);
-      SmartDashboard.putString("CDS Wheel Controller Direction", "Forward");
-      SmartDashboard.putNumber("CDS Wheel Controller Speed", Constants.CDSWheelControllerSpeed);
```

Comment:
Keep for intake team debugging

Reply:
Intake team debugging can use the NetworkTable, it's basically the same but only on a separate tab